src/backend/InvenTree/InvenTree/fields.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, **kwargs):
 
     def run_validation(self, data=empty):
         """Override default validation behaviour for this field type."""
-        strict_urls = get_global_setting('INVENTREE_STRICT_URLS', True, cache=False)
+        strict_urls = get_global_setting('INVENTREE_STRICT_URLS', cache=False)
```

Comment:
Q: Why is this changed?

Reply:
The default value filters through from the definition of the setting itself, and does not need to be passed here.